webpack.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
             }
         }, {
             test: /\.js$/,
-            include: /node_modules[\\\/](buffer|d3-color|d3-interpolate|is-mobile)[\\\/]/,
+            include: /node_modules[\\\/](buffer|d3-color|d3-interpolate|is-mobile|chart2music)[\\\/]/,
```

Comment:
This change unfortunately doesn't do anything to resolve the build check issues. Is there something else needed?

Reply:
This still helps converting `chart2music` dependency to es5. So let's keep it here.
The build issue you are mentioning is related to new file `src/plot_api/accessibility.js` being written in es6.
@alexcjohnson wondering what `webpack` configuration we used in other libraries to bundle es6 files in `src`?

Reply:
When I write that file in es5 using require instead of import, it can't find the chart2music module. I have tried putting type 'module' in the package.json. I think maybe something to do with chart2music not being a .mjs file